styles.css

```diff
@@ -301,11 +313,78 @@ li {
 }
 
 .contact-container {
-  margin-top: 150px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   border: 2px solid #6070ff;
   border-top-left-radius: 100px;
   background-color: #6070ff;
-  height: 100vw;
-  position: relative;
-  margin-left: 0px 0px;
-}
\ No newline at end of file
+  height: auto;
+  box-sizing: border-box;
+  margin: -20px;
+  margin-top: 150px;
+}
+
+.contact-container input {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  height: 48px;
+  width: 273px;
+  margin-top: 15px;
+}
+
+.contact-container h1 {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-container p {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin-top: -20px;
+}
+
+.contact-container textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 273px;
+  height: 170px;
+  margin-top: 18px;
+}
+
+.contact-container button {
+  width: 140px;
+  height: 48px;
+  border-radius: 8px;
+  background: #fff;
+  color: #6070ff;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  margin-top: 22px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 60px;
+  border: 2px solid grey;
+  padding: 12px 16px;
+}
+
+button:hover {
+  color: #00f;
+  background-color: #000;
+}
```

Comment:
- [ ] Also make sure your button has the `hover`, `active`, `disable` state as it is stated in the template
![image](https://user-images.githubusercontent.com/97100091/212951228-24985255-4ece-455b-90a6-b4906e289e6b.png)